lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -116,7 +116,8 @@ def verify(credit_card, options = {})
         add_payment_type(post, options)
         customer_country(post, options)
         post[:card] = payment_details(credit_card)
-        post[:device_id] = options[:device_id] if options[:device_id]
+        post[:payment] = {}
```

Comment:
Just a quick gut check on this - I notice that unlike the `purchase` and `authorize` methods, the `post` on `verify` doesn't get instantiated with `post[:payment] = {}`, which is why you had add that here.

Are we sure from the EBANX docs that the `payment` object should be part of the verify operation too? Just curious about this since nothing else in this transaction type is nested within 🤔